src/main/java/xbot/common/command/BaseMaintainerCommand.java

```diff
@@ -207,11 +207,6 @@ protected boolean getErrorWithinTolerance() {
      */
     protected abstract double getHumanInputMagnitude();
 
-    @Override
-    public String getPrefix() {
-        return subsystemToMaintain.getPrefix() + getName() + "/";
```

Comment:
This just fixes a nullref, since the maintainer command will be created before the subsystem to maintain is set. On the upside, at least the records for this class will be just like other commands.